scikeras/wrappers.py

```diff
@@ -1626,6 +1626,9 @@ def r_squared(y_true, y_pred):
         """A simple Keras implementation of R^2 that can be used as a Keras
         loss function.
 
+        Note that this returns 1-R^2 so that it can be minimized as a loss
+        function.
```

Comment:
I'm not sure I'd include this line. I think saying that R^2 is coefficient of determination suffices. Maybe you can say that R^2 is an equivalent measure to MSE (specifically with `mse = (1 - coef) * c` for some constant `c`).

Reply:
I resolved the comment from the docstring as per https://github.com/adriangb/scikeras/pull/189#discussion_r576321029. Can I resolve the comment?